docs/core/testing/unit-testing-mstest-migration-from-v3-to-v4.md

```diff
@@ -0,0 +1,199 @@
+---
+title: MSTest migration from v3 to v4
+description: Learn about migrating to MSTest v4.
+author: Youssef1313
+ms.author: ygerges
+ms.date: 07/22/2025
+---
+
+# Migrate from MSTest v3 to MSTest v4
```

Comment:
I don't know if we should write down some reasoning as to why we are doing these breaks.